samples/ImageScanning/cs/ImageScanningSample_UWP/CustomControls/CurvedEdgesDetectorInteractiveControl.xaml

```diff
@@ -0,0 +1,46 @@
+﻿<UserControl
+    x:Class="ImageScanningSample.CurvedEdgesDetectorSkillInteractiveControl"
+    xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
+    xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
+    xmlns:local="using:ImageScanningSample"
+    xmlns:sys="using:System"
+    xmlns:controls="using:Microsoft.Toolkit.Uwp.UI.Controls"
+    xmlns:d="http://schemas.microsoft.com/expression/blend/2008"
+    xmlns:mc="http://schemas.openxmlformats.org/markup-compatibility/2006"
+    mc:Ignorable="d"
+    d:DesignHeight="300"
+    d:DesignWidth="400">
+    <StackPanel Orientation="Horizontal">
+        <controls:Expander Header="Input values" IsExpanded="True">
+
+            <StackPanel>
+                <TextBlock Text="Input Quad:" />
+                <StackPanel Orientation="Horizontal" >
+                    <TextBlock Text="X: " />
+                    <TextBox Name="UITopLeftBaseQuadCornerX" IsReadOnly="True" IsEnabled="False"/>
```

Comment:
Just curious, why use a `TextBox` instead of a `TextBlock` here?

Reply:
preferred the way the value is displayed